default-config.json

```diff
@@ -60,6 +60,15 @@
         "    2. Provided the name of a command (`%helpTopic help`), it will provide details regarding the specific usage of that command."
       ]
     },
+    "nowplaying": {
+      "internalKey": "nowplaying",
+      "lines": [
+        "The `%nowplaying` command displays the song that is currently playing on Cadence."
```

Comment:
"The `Cadence now playing` command..."

Hmm. This makes me feel like I have a design flaw here - We've been using internal names for this: 'The `nowplaying` command displays the song that is currently playing on Cadence.'

But you have a point here that that's a little awkward... I had this same thought for aliases, when I was considering the 'Specialized alias-listing command' route - The internal key would be a bad thing to use as the trigger phrase (`Cadence aliases nowplaying`) because it's by definition internal - Even though internal keys are mostly intuitive ("The play command") there's a philosophical point to not requiring the user to understand them - For example if I reconfigure CadenceBot to accept commands in Spanish, it would be stupid to say `Cadence busca`, but also `Cadence alias search`.

So I was thinking there should be a command friendlyNames or some such - A short string that we can refer to the command as, without it being the command phrase (Because I feel like there's a better way to say 'The Cadence now playing command'). Of course in default config these will be mostly the same as the internal keys:

play => play
stop => stop
help => help
helpTopic => help
search => search
request => request
library => library
nowplaying => now playing

But this provides a layer we could use to abstract our userspace away from our implementation details.

We can then expose this to helptext similar to the trigger phrase - Perhaps something like `%friendly play`?

So this line then becomes 'The `%friendly nowplaying` command...', and prints 'The `now playing` command...'?

What do you think? If you don't want to implement this yourself we can leave this PR as-is and I'll handle it (Or, if you want, we could split it into its own issue that you take next...)

Reply:
I see what you're saying. I'm not sure what's the best syntax is for doing a non-prefixed command shorthand but I'd prefer to pull it out to a different PR and handle it there (which I'm fine doing ;))

Reply:
Yes - Let's file a new issue and discuss it separately. Feel free to assign it to yourself if you want it! We'll leave this as-is for now.